Sources/Plasma/PubUtilLib/plPipeline/GL/plGLMaterialShaderRef.cpp

```diff
@@ -659,7 +659,9 @@ void plGLMaterialShaderRef::ILoopOverLayers()
         fFragmentShader->PushFunction(fragPass);
 
         std::shared_ptr<plConditionNode> passCond = COND(IS_EQ(uPass, CONSTANT(ST::format("{}", pass))));
-        passCond->PushOp(CALL(fragPass->name));
+        
+        std::shared_ptr<plOutputVariableNode> output = IFindVariable<plOutputVariableNode>("fragColor", "vec4");
+        passCond->PushOp(ASSIGN(output, CALL(fragPass->name)));
```

Comment:
Changing the shader to set an out vec4 called frag color. This doesn't enforce that fragColor should be in the first output position, which modern OpenGL says it should be to act as the return value.